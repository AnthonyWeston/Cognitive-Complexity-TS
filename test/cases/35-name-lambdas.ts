function f() {
    const ff = () => {};

    const fff = function() {}

    const g = () => {
        new Promise(() => {

        });
    };

    const h = function() {
        new Promise(() => {

        });
    };

    new Promise(() => {

    });
}
