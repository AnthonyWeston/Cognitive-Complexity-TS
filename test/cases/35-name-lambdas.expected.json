{
    "35-name-lambdas.ts": {
        "score": 0,
        "inner": [
            {
                "name": "f",
                "score": 0,
                "line": 1,
                "column": 1,
                "inner": [
                    {
                        "name": "ff",
                        "score": 0,
                        "line": 2,
                        "column": 16
                    },
                    {
                        "name": "fff",
                        "score": 0,
                        "line": 4,
                        "column": 17
                    },
                    {
                        "name": "g",
                        "score": 0,
                        "line": 6,
                        "column": 15,
                        "inner": [
                            {
                                "name": "",
                                "score": 0,
                                "line": 7,
                                "column": 21
                            }
                        ]
                    },
                    {
                        "name": "h",
                        "score": 0,
                        "line": 12,
                        "column": 15,
                        "inner": [
                            {
                                "name": "",
                                "score": 0,
                                "line": 13,
                                "column": 21
                            }
                        ]
                    },
                    {
                        "name": "",
                        "score": 0,
                        "line": 18,
                        "column": 17
                    }
                ]
            }
        ]
    }
}
